### PR TITLE
Add olive-writing manual

### PR DESCRIPTION
**Pull request by callunity**
A note: All of the `the folder where the Shesmu JAR lives` parts don't apply to our custom Shesmu deployment, but I was trying to write from the perspective of someone who is setting up a new Shesmu.

### CODE REVIEW

File: olive-writing.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+### JIRA config files
+Each project in your JIRA must be configured using a separate file named `<project-name>.jira` with the following format:
+```
+{
+  "projectKey": <code for project>,
+  "token": <API token for project>,
+  "url": <URL of the JIRA instance>
+}
+```
+If too many JIRA actions are launched at once, JIRA may (blacklist? throttle?) more actions from that API key. This can be avoided by adding in a `jira.ratelimit` file to the Shesmu server folder, which tells Shesmu to limit the number of JIRA actions it launches. The following `jira.ratelimit` example file lists some useful defaults:
```
Comment (apmasell) [inline olive-writing.md]: I don't think JIRA blacklists you, I think it just crashes or makes IT sad.

File: olive-writing.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+```
+{
+  "capacity": 50,
+  "delay": 500
+}
+```
+
+### Action files
```
Comment (apmasell) [inline olive-writing.md]: This only applies if you have all the Guanyin infrastructure plus a bunch of stuff that we haven't released publically...

File: olive-writing.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    [...filter(s) for what conditions should cause an open ticket to be closed...]
+  Run resolve_ticket_<project-name> With
+    summary = <ticket summary/title>,
+    comment = <explanation for why ticket can now be closed>;
+```
+Note that `<project-name>` must be the same name as the `<project-name>.jira` file located in the Shesmu server directory, and the `<ticket summary/title>` must be identical to the `<ticket summary/title>` in the olive which opened the ticket.
+
+
+
+## Write olives for custom actions
```
Comment (apmasell) [inline olive-writing.md]: Guanyin reports

File: olive-writing.md (hunk 1 above)
Comment (morgantaschuk) [reply olive-writing.md]: No, it will lock your account and make you do a captcha before you can log in again.

File: launch-actions.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+## Test an olive
+End your olive with `Run nothing;`. This will cause no actions to be launched
+until the correctness of the olive's behaviour is confirmed.
+Start the Shesmu server and navigate to http://localhost:8081 .
+
+1. Check the [Status](http://localhost:8081) page. Any olive compilation errors
+	 will be displayed beneath the **Core** section at the top of the page.
+1. Use the Olives dashboard to determine how much data is flowing through each node.
+		* The left column of an olive diagram is the data flow count. It tracks the
```
Comment (apmasell) [inline launch-actions.md]: These bullets don't seem to display correctly in github.

**Pull request merged**